src/vocabularyChecker/detectWrongSyntax.test.ts

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import {checkInput, transformTextToStringArray, eliminatePunctuation} from "./detectWrongSyntax";
+
+
+it("Works for simple text", () => {
+    const wrongWords = checkInput("fu bar bloedsinn"); //fu and bar are indeed included in allowedVocab
+    expect(wrongWords).toEqual(["bloedsinn"]);
+  });
+
+it("Works for an array containing only the empty String", () =>{
```

Comment:
Generally it's nice if the text explains the behavior in a bit more detail than "it works" - so for this it'd be something like `it("outputs an empty array when input contains only the empty string"...)` or `it("detects the empty string as invalid word"...)` depending on what the expected behavior is. Then a user can simply scan the tests to understand what the functionality is, and doesn't have to look at the test implementation to find out what the expected behavior is.